languages/en.php

```diff
@@ -619,35 +619,35 @@
 
 	'admin:options' => 'Admin options',
 
-	'admin:administer_security' => 'Security',
```

Comment:
How could we avoid changing all these language keys? Maybe a little inline function to rewrite keynames?